meson.build

```diff
@@ -10,6 +10,7 @@ conf.set_quoted('PACKAGE_VERSION', meson.project_version())
 
 cc = meson.get_compiler('c')
 objcopy = find_program('objcopy')
+objcopy_version = run_command(objcopy, '--version').stdout().split('\n')[0].split(' ')[-1]
```

Comment:
works for me, but looks kinda scary...

Reply:
Any suggestions for making it prettier?
It seems like  --version has lots of extraneous text.